README.md

```diff
@@ -1,61 +1,44 @@
-# emis-web
+# ewea-web
 
 [![Build Status](https://travis-ci.org/CodeTanzania/emis-web.svg?branch=develop)](https://travis-ci.org/CodeTanzania/emis-web)
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 
 <img src="docs/images/logo.svg" 
-alt="EMIS Logo" width="100" height="auto" />
+alt="EWEA Logo" width="100" height="auto" />
 
-## About EMIS
+## About EWEA
 
-A collaboration platform that facilitates communication, planning and actions for disaster management.
+EWEA web is an Early Warning, Early Action tool that facilitates the disseminationn of warnings, communication and actions associated with the warnings.
 
-It incorporates data that can be shared and analyzed in order generate insights for local authorities and communities to better manage disasters and increase resilience to disaster risk. The data is organized in core components.
+It that can be shared and analyzed in order generate insights for local authorities and communities to better manage disasters and increase resilience to disaster risk. The data is organized in core components.
 
-![EMIS Homepage](docs/images/home.png 'EMIS Homepage')
+![EWEA Homepage](docs/images/home.png 'EWEA Homepage')
 
-## EMIS Components
+## EWEA Components
 
 #### Stakeholders
 
 EMIS Stakeholders contains an up to date database of all stakeholders responsible for emergency/disaster management within a specific region.
 
 See source code [repository](https://github.com/CodeTanzania/emis-stakeholder).
 
-#### Resources
-
-EMIS Resources depends heavily on EMIS Stakeholders as it acts as a resource management tool for resources owned by stakeholders.
-
-See source code [repository](https://github.com/CodeTanzania/emis-resource).
 
 #### Alerts
 
-This component consists of an up to date database of ingested emergency / disaster alerts from multiple sources in near real time and disseminating them to disaster management stakeholders.
+EWEA alerts consists of alerts created This component consists of an up to date database of ingested emergency / disaster alerts from multiple sources in near real time and disseminating them to disaster management stakeholders.
 
 See source code [repository](https://github.com/CodeTanzania/emis-alert).
 
 #### Emergency Plans
 
-EMIS Plans provides a way to create, store and disseminate emergency/disaster management plans with detailed activities and Standard Operating Procedures (SOPs) to be performed by each stakeholder in all phases of disaster management.
+EMIS Plans consists of Standard Operating Procedures (SOPs) that provide a guide for early actions that accompany the early warnings.
```

Comment:
This still EMIS here .. Plus now we don't have plan

Reply:
Resoled in 1324e4b